ui/client/types/definition.ts

```diff
@@ -7,17 +7,28 @@ export type UIObjectDefinition = {
     nodeConfig: SingleNodeConfig,
 }
 
-export type TypingResult = $TodoType
+export type TypingResult = {
```

Comment:
This is not correct definition, I think. There is no "name" field, params and refClazzName are not present in all cases (e.g. union or dict instance)?

Reply:
Yes, you're right, it should be defined this way:
```
TypingResult = {
    display: string,
    type: string,
    refClazzName?: string,
    params?: TypingResult[]
}
```

We could also type this as a union, but it would entail some cumbersone changes in few places

Reply:
@mproch 
https://github.com/TouK/nussknacker/pull/1191/commits/03b42f65903ce8692a19ee6c1a9dc122cb237ce5

Reply:
Maybe some comment about union handling? (that is - that we don't handle it ATM)